specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -606,6 +623,99 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/operationStatus/{operationId}": {
+      "get": {
+        "tags": [
+          "Vaults"
+        ],
+        "description": "Gets the operation status for a resource.",
+        "operationId": "GetOperationStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
```

Comment:
Missing error response model. Consider using error models defined in types.json in common-types folder: [https://github.com/Azure/azure-rest-api-specs/tree/master/specification/common-types/resource-management](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/common-types/resource-management)

Reply:
Yeah, this is actually an issue on a lot of the responses in this swagger.

---
In reply to: [551566447](https://github.com/Azure/azure-rest-api-specs/pull/12248#discussion_r551566447) [](ancestors = 551566447)